src/components/sidePanel/SidePanel.js

```diff
@@ -1,17 +1,35 @@
-import React from 'react'
-import './SidePanel.css'
+import React from 'react';
```

Comment:
Added my own page as a link and ran `npm run lint`. No other changes were made here.